api/game.py

```diff
@@ -57,6 +58,8 @@ def get_role_description(role):
         "Maelagant": "You may play Reversal cards while on missions. \nLike other Evil characters, you know who else is Evil (except Colgrevance).",
         "Agravaine": "You must play Fail cards while on missions. \nIf you are on a mission that Succeeds, you may declare as Agravaine to cause it to Fail instead. \nLike other Evil characters, you know who else is Evil (except Colgrevance).",
         "Colgrevance": "You know not only who else is Evil, but what role each other Evil player possess. \nEvil players know that there is a Colgrevance, but do not know that it is you.",
+        "Jealous Ex": "You see either Iseult (if there are two lovers) or the Older Sibling (if there are no lovers).\nIf Tristan correctly identifies you at the end of the game, Tristan and Iseult will receive immunity from assassination.",
```

Comment:
It's not super clear to me what the end game looks like. After good wins, does Tristan reveal if they want to make a guess at the jealous ex (if it exists) and then guess? Can Tristan just stay silent to pretend that Iseult is an older sibling? I guess in the world that they were too obvious, they could hail mary a guess...